crates/optimism/txpool/src/metrics.rs

```diff
@@ -0,0 +1,20 @@
+//! Optimism transaction pool metrics
+
+use reth_metrics::{metrics::Histogram, Metrics};
+use std::time::Duration;
+
+/// Optimism transaction pool metrics
+#[derive(Metrics, Clone)]
+#[metrics(scope = "optimism_transaction_pool")]
+pub struct OpTxPoolMetrics {
```

Comment:
```suggestion
pub struct SupervisorMetrics {
```
name can be more specific to its use case